jsonnet/openebs-mixin/rules/jiva-volume-rules.libsonnet

```diff
@@ -0,0 +1,30 @@
+function(param) {
```

Comment:
Basic question: can we combine jiva-rules.libsonnet and jiva-volume-rules.libsonnet into one?

Reply:
Yes, we can.